src/lib/operators/pqp_expression.cpp

```diff
@@ -100,7 +100,7 @@ bool PQPExpression::operator==(const PQPExpression& other) const {
   return _column_id == other._column_id;
 }
 
-std::shared_ptr<PQPExpression> PQPExpression::set_placeholder_value(const AllTypeVariant& value) {
+std::shared_ptr<PQPExpression> PQPExpression::copy_with_placeholder_value(const AllTypeVariant& value) {
```

Comment:
Sorry to point this out in this PR, but what is this function doing? All it is doing is re-using the alias and the _column_id from this expression? It's not even Asserting this Expression is actually a placeholder.
I'd expect a method with this name to recursively search the expression for placeholders and replace those subexpression with placeholders... I think I'll create an issue for this.

Reply:
#780